test/uk/gov/hmrc/cdsreimbursementclaim/controllers/DeclarationInfoControllerSpec.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaim.controllers
+
+import cats.data.EitherT
+import play.api.http.{HeaderNames, Status}
+import play.api.libs.json.{JsObject, JsString, JsValue}
+import play.api.test.Helpers._
+import play.api.test._
+import uk.gov.hmrc.cdsreimbursementclaim.config.AppConfig
+import uk.gov.hmrc.cdsreimbursementclaim.models.Error
+import uk.gov.hmrc.cdsreimbursementclaim.services.DeclarationInfoService
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+
+import scala.concurrent.Future
+
+class DeclarationInfoControllerSpec extends ControllerSpec with DefaultAwaitTimeout {
+
+  implicit val ec            = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Would be good to put this outside the spec like you did with the TPI05 controller spec